components/k8s-model-server/http-proxy/server.py

```diff
@@ -125,25 +148,27 @@ def decode_b64_if_needed(data):
   else:
     return data
 
+
 def get_signature_map(model_server_stub, model_name):
   """ Gets tensorflow signature map from the model server stub.
 
-  Args:
-    model_server_stub: The grpc stub to call GetModelMetadata.
-    model_name: The model name.
+    Args:
+      model_server_stub: The grpc stub to call GetModelMetadata.
+      model_name: The model name.
 
-  Returns:
-    The signature map of the model.
+    Returns:
+      The signature map of the model.
   """
   request = get_model_metadata_pb2.GetModelMetadataRequest()
   request.model_spec.name = model_name
   request.metadata_field.append("signature_def")
   try:
-    response = model_server_stub.GetModelMetadata(request, MODEL_SERVER_METADATA_TIMEOUT_SEC)
+    response = model_server_stub.GetModelMetadata(
+      request, MODEL_SERVER_METADATA_TIMEOUT_SEC)
```

Comment:
I would have preferred request to be indented 4 because of hanging indention.

Reply:
With the recent update it does:
```
  try:
    response = model_server_stub.GetModelMetadata(
        request, MODEL_SERVER_METADATA_TIMEOUT_SEC)
```
So we should be good.